framework/core/components/backend.php

```diff
@@ -747,6 +747,37 @@ public function _action_create_taxonomy_options( $term ) {
 		echo $this->render_options( $collected, $values, array(), 'taxonomy' );
 	}
 
+	/**
+	 * @param string $taxonomy
+	 */
+	public function _action_create_add_taxonomy_options( $taxonomy ) {
+		$options = fw()->theme->get_taxonomy_options( $taxonomy );
+		if ( empty( $options ) ) {
+			return;
+		}
+
+		$collected = array();
+
+		fw_collect_options( $collected, $options, array(
+			'limit_option_types'    => false,
+			'limit_container_types' => array(), // only simple options are allowed on taxonomy edit page
+			'limit_level'           => 1,
+		) );
+
+		if ( empty( $collected ) ) {
+			return;
+		}
+		$values = array();
+		foreach ( $options as $option_id => $values ) {
+			$values[ $option_id ] = $values['value'];
```

Comment:
This is non sense. On options render default values will be extracted automatically